templates/contact-us/index.html

```diff
@@ -0,0 +1,77 @@
+{% extends "contact-us/base_contact-us.html" %}
+
+{% block title %}Contact Canonical{% endblock %}
+
```

Comment:
Do we need the meta description here?
`Ubuntu is an open source software platform that runs everywhere from the smartphone, the tablet and the PC to the server and the cloud.`

Reply:
That's a generic one... and bad as we don't do smartphones and tablets... I added a better one for contact-us and updated the copy doc